task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -67,6 +67,7 @@
 )
```

Comment:
Can we also add a task trying to access inlet events in here? Example: `test_run_with_asset_outlets`

Reply:
sure, just added